### PR TITLE
test: test mp_guess_lang_from_filename with [] and ()

### PR DESCRIPTION
**Pull request by guidocella**

### CODE REVIEW

File: test/language.c [hunk 1]
```diff
@@ -76,5 +76,10 @@ int main(void)
     assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo.en-aaaaaaaaa.srt"), NULL)), "");
     assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo.en-0.srt"), NULL)), "");
 
+    assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo[en].srt"), NULL)), "en");
+    assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo[en-US].srt"), NULL)), "en-US");
+    assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo(en).srt"), NULL)), "en");
+    assert_string_equal(bstrto0(ta_ctx, mp_guess_lang_from_filename(bstr0("foo(en-US).srt"), NULL)), "en-US");
```
Comment (kasper93) [inline test/language.c:82]: add also test for empty `[]` and `()`

**Review by kasper93 — APPROVED** [review]
